refinery/ui/source/js/file-browser/partials/assay-filters.html

```diff
@@ -54,7 +54,9 @@ <h4 class="panel-title">
         <div class="panel-body">
           <div ng-repeat="field in attributeObj.facetObj | orderBy: order track by $index">
             <div class="checkbox container"
-                ng-show="field.name.toLowerCase().includes(search.toLowerCase()) || showField(field.name, attributeObj.internal_name, attribute)">
+                ng-show="(search.toLowerCase().length && field.name
```

Comment:
or just `search.length`? And maybe only line break on the `||`... or that might be too long? Thanks for fixing this / sorry for breaking it.